### ISSUE TITLE
Config not found

### ISSUE THREAD

**Issue by nichtich** [peggyjs/peggyjs-eslint-plugin#35]
Following the instruction with [a minimal example](https://github.com/nichtich/test-peggy-eslint) I get this error message:

~~~
> lint
> eslint --fix '*.peggy'

{ all: undefined, recommended: undefined }

Oops! Something went wrong! :(

ESLint: 8.57.0

ESLint couldn't find the config "plugin:@peggyjs/recommended" to extend from. Please check that the name of the config is correct.
~~~

**Comment by hildjj** [comment]
OK, I'm not sure how I broke this, but I did.  willfix.

**Comment by hildjj** [comment]
Fixed in v2.1.1
